### PR TITLE
Update Event structure

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**: Updates Event structure.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: n/a

**Special notes for your reviewer**: n/a

**Documentation**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/api/v1/types.go [hunk 1]
```diff
@@ -674,4 +676,11 @@ type Event struct {
 
 	// The object name that those events are about.
 	InvolvedObjectName string `json:"involvedObjectName"`
+
+	// The time at which the most recent occurrence of this event was recorded.
+	// swagger:strfmt date-time
```
Comment (p0lyn0mial) [inline api/pkg/api/v1/types.go:681]: wow :)

File: api/pkg/api/v1/types.go [hunk 2]
```diff
@@ -674,4 +676,11 @@ type Event struct {
...
+	LastTimestamp Time `json:"lastTimestamp,omitempty"`
```
Comment (p0lyn0mial) [inline api/pkg/api/v1/types.go:682]: oh, wait shouldn't we use our `Time` struct ? https://github.com/kubermatic/kubermatic/blob/a5dc4f75d0757d789f712aeb19e582e1859efca2/api/pkg/api/v1/time.go

**Comment by p0lyn0mial** [comment]
/hold

not sure about `Time` struct.

File: api/pkg/api/v1/types.go (hunk 2 above)
Comment (maciaszczykm) [reply api/pkg/api/v1/types.go:682]: That's the one I am using. It is in the same package :)

Comment (p0lyn0mial) [reply api/pkg/api/v1/types.go:682]: I must have checked a wrong file :)

**Pull request merged**